packages/core/src/util/sequelize.ts

```diff
@@ -9,12 +9,52 @@ export const dbCredsSchema = z.object({
   port: z.number(),
   engine: z.literal("postgres"),
 });
+export type DbCreds = z.infer<typeof dbCredsSchema>;
+
+export const dbReadReplicaEndpointSchema = z.object({
+  host: z.string(),
+  port: z.number(),
+});
+export type DbReadReplicaEndpoint = z.infer<typeof dbCredsSchema>;
```

Comment:
Shouldn't the correct line here be:
export type DbReadReplicaEndpoint = z.infer<typeof dbReadReplicaEndpointSchema>;.

Reply:
I don't see what you mean @jonahkaye . The code on your comment doesn't seem right.

Reply:
The `DbReadReplicaEndpoint` type is being inferred from the `dbCredsSchema` and not the `dbReadReplicaEndpointSchema`

Reply:
Oh, good catch